### ISSUE TITLE
`fitz.__version__`` does not work anymore

### ISSUE THREAD

**Issue by noppelmax** [pymupdf/PyMuPDF#3774]
### Description of the bug

In v1.23.9 I was able to do `fitz.__version__` like in any other lib. In v1.24.9 this does no longer work. Is that intended?

```
max@merry ~/github/zotlinksToPdfs                                                    [14:05:14]
(base) > $ python demo.py                                                                      
Traceback (most recent call last):
  File "/home/max/github/zotlinksToPdfs/demo.py", line 3, in <module>
    print(fitz.__version__)
          ^^^^^^^^^^^^^^^^
AttributeError: module 'fitz' has no attribute '__version__'
```

### How to reproduce the bug

```python
import fitz  # PyMuPDF

print(fitz.__version__)
```

### PyMuPDF version

1.24.9

### Operating system

Linux

### Python version

3.11

**Comment by JorjMcKie** [comment]
Thanks for reporting this!
Please use PyMuPDF as the import name, or `import pymupdf as fitz`.
The alias "fitz" may not have copied over one or the other original attributes.
And we do not want to chase every single one of such minor issues.

**Comment by noppelmax** [comment]
Great! That works! Thank you!

**Comment by julian-smith-artifex-com** [comment]
Fixed in 1.24.10.
